### PR TITLE
NOT A PR: Just doing a 120 reps of desktop/signs_up.feature:56 on Travis

### PR DESCRIPTION
**Pull request by FinnWoelm**
**THIS IS NOT A PULL REQUEST -- IT IS JUST INTENDED TO RUN 120 TESTS ON TRAVIS**

I have a fix for randomly failing feature desktop/signs_up.feature:56. I've confirmed it on my fork but I want to create a record of the "evidence" here, too.

This branch sets up Travis to test a 120 repetitions of features/desktop/signs_up.feature:56. This will provide us with sufficient data from CI to determine whether or not the fix is working.

In this pre-fix Travis run, we should see about 25-33% of features fail (between 30-40 out of the 120). After Travis has run this code once, I will push a third commit that includes the fix. Then we will have another 120 reps of signs_up.feature:56 and we can compare results.

In a minute, I will also open a PR that includes just the fix. That PR leaves all other files intact, so we will have output from the entire cucumber suite. That PR is the one to be considered for merging.

### CODE REVIEW

File: app/assets/javascripts/app/views/publisher/poll_creator_view.js [hunk 1]
```diff
@@ -91,18 +91,20 @@ app.views.PublisherPollCreator = app.views.Base.extend({
   validatePoll: function() {
     var _this = this;
     var inputs = this.$('input:visible');
+    console.log("Validating " + inputs.length + " poll inputs...");
```
Comment (diaspora-code-review) [inline app/assets/javascripts/app/views/publisher/poll_creator_view.js]: Unexpected console statement.

File: app/assets/javascripts/app/views/publisher/poll_creator_view.js [hunk 2]
```diff
@@ -91,18 +91,20 @@ app.views.PublisherPollCreator = app.views.Base.extend({
...
     _.each(inputs, function(input, i){
       // Validate the input unless it is the last one, or there are only the
       // question field and two options
       if( i !== inputs.length - 1 || inputs.length <= 3) {
         if(_this.validateInput($(input)) === false) pollValid = false;
-      }      
+      }
     });
 
+    console.log("Poll is" + (pollValid ? '' : ' not') + " valid");
```
Comment (diaspora-code-review) [inline app/assets/javascripts/app/views/publisher/poll_creator_view.js]: - Unexpected console statement.
- Strings must use doublequote.

File: app/assets/javascripts/app/views/publisher_view.js [hunk 3]
```diff
@@ -182,6 +182,9 @@ app.views.Publisher = Backbone.View.extend({
   },
 
   createStatusMessage : function(evt) {
+
+    console.log("'Post' button has been clicked...");
```
Comment (diaspora-code-review) [inline app/assets/javascripts/app/views/publisher_view.js]: Unexpected console statement.

File: app/assets/javascripts/app/views/publisher_view.js [hunk 4]
```diff
@@ -477,6 +480,12 @@ app.views.Publisher = Backbone.View.extend({
         isPhotoAttached = (this.photozoneEl.children().length > 0),
         isValidPoll = this.viewPollCreator.validatePoll();
 
+    console.log("Checking if post can be submitted:");
```
Comment (diaspora-code-review) [inline app/assets/javascripts/app/views/publisher_view.js]: Unexpected console statement.

File: app/assets/javascripts/app/views/publisher_view.js [hunk 5]
```diff
@@ -477,6 +480,12 @@ app.views.Publisher = Backbone.View.extend({
...
+    console.log("Post has text or photo: " + (!onlyWhitespaces||isPhotoAttached));
```
Comment (diaspora-code-review) [inline app/assets/javascripts/app/views/publisher_view.js]: - Unexpected console statement.
- Infix operators must be spaced.

File: app/assets/javascripts/app/views/publisher_view.js [hunk 6]
```diff
@@ -477,6 +480,12 @@ app.views.Publisher = Backbone.View.extend({
...
+    console.log("Post poll does not exist or is invalid: " + (isValidPoll));
```
Comment (diaspora-code-review) [inline app/assets/javascripts/app/views/publisher_view.js]: Unexpected console statement.

File: app/assets/javascripts/app/views/publisher_view.js [hunk 7]
```diff
@@ -477,6 +480,12 @@ app.views.Publisher = Backbone.View.extend({
...
+    var canSubmit = (!onlyWhitespaces || isPhotoAttached) && isValidPoll && !this.disabled
```
Comment (diaspora-code-review) [inline app/assets/javascripts/app/views/publisher_view.js]: Missing semicolon.

File: app/assets/javascripts/app/views/publisher_view.js [hunk 8]
```diff
@@ -477,6 +480,12 @@ app.views.Publisher = Backbone.View.extend({
...
+    console.log("Post is" + (canSubmit ? '' : ' not') + " submittable.")
```
Comment (diaspora-code-review) [inline app/assets/javascripts/app/views/publisher_view.js]: - Unexpected console statement.
- Strings must use doublequote.
- Missing semicolon.

**Comment by fwoelm** [comment]
The pre-fix tests on Travis have resulted in **35 failures** (29%) and **85 successes** (71%).

I have just pushed the fix, so let's see how Travis does now!

**Comment by fwoelm** [comment]
Uhm, not sure why Travis is refusing to run this again... I'll push an empty file just to see if I can get it to re-run with the fix.

**Comment by fwoelm** [comment]
Hmm, I think Travis is still reluctant. Maybe one of you can trigger it manually. Or you can also see that post-fix results are all green on my own fork: https://github.com/UpshiftNetwork/diaspora/pull/11

**Comment by Flaburgan** [comment]
I restarted the travis jobs.

**Comment by fwoelm** [comment]
@Flaburgan Thanks! But you restarted it with the pre-fix commits :D I might just have to do a new PR with the fix...

**Comment by Flaburgan** [comment]
Honestly, https://github.com/diaspora/diaspora/pull/7497 is green, that looks good enough to me. As far as there is no regression with your fix, it can't hurt, so we're fine. I'm cancelling the job here, we're trying to limit as much as we can the usage of travis resources.

**Comment by fwoelm** [comment]
If it's good to you, it's good to me, too. Closing this! :)

**Comment by Flaburgan** [comment]
@fwoelm btw for "real time" discussions about contribution you can come on #diaspora-dev on IRC Freenode. It's easier ;)

**Pull request closed without merging**